File: Java/Program/Shirt.java
public class Shirt extends Clothing{
    String color, sleeveType;

    public Shirt(){
        this.color = "";
        this.sleeveType = "";
    }

    public Shirt(String color, String sleeveType){
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }

    public String getColor() {
        return color;
    }

    public String getSleeveType() {
        return sleeveType;
    }
}

File: PHP/Program/Shirt.php
<?php
require("Clothing.php");

class Shirt extends Clothing{
    private $color;
    private $sleeveType;

    public function __construct($color = "", $sleeveType = ""){
        $this->color = $color;
        $this->sleeveType = $sleeveType;
    }

    public function getColor(){
        return $this->color;
    }
    public function getSleeveType(){
        return $this->sleeveType;
    }

    public function setColor($color){
        $this->color = $color;
    }
    public function setSleeveType($sleeveType){
        $this->sleeveType = $sleeveType;
    }
}
?>